README.md

```diff
@@ -2,6 +2,15 @@
 
 Repository for out-of-tree scheduler plugins based on the [scheduler framework](https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/).
 
+This repository includes some incubating ideas of solveing real problems which is hard to be solved with only in-tree scheduler plugins. 
```

Comment:
As you said, there is no clear criteria. So we can't call them incubating ideas.

I prefer to call it an enhanced scheduler based on the scheduler framework